### ISSUE TITLE
Kitty is ignoring my config on MacOS

### ISSUE THREAD

**Issue by luisdavim** [kovidgoyal/kitty#1331]
I have my config in `~/.config/kitty/kitty.conf` and `~/Library/Preferences/kitty` is a symbolic link to it but kitty ignores my configs.

The only way for it to load my configs is to start kitty from another terminal with `/Applications/kitty.app/Contents/MacOS/kitty -c ~/.config/kitty/kitty.conf`

**Comment by kovidgoyal** [comment]
I cannot replicate on my macOS machine, kitty reads the config from
~/.config/kitty/kitty.conf

I suggest removing the symlink, it is not needed. And if that does not
work, use the kitty command line facility to pass the --config option
permanently, see the kitty FAQ for how to use this facility.

**Comment by luisdavim** [comment]
that didn't work for me

**Comment by magicalbanana** [comment]
This is the same for me. Any thoughts @kovidgoyal? I am OSX Mojave.

**Comment by antoinesauray** [comment]
I have the same problem using the MacOS launcher. If I start Kitty from another terminal (including Kitty itself) the configuration is correctly applied.

**Comment by Luflosi** [comment]
Are there any messages in Console.app that could be related? You can filter the messages to only include the process `kitty`.

**Comment by antoinesauray** [comment]
I solved it actually. There was a background kitty process that was running. I killed it and then the configuration was applied correctly.

**Comment by saurap** [comment]
I found out that I need to restart kitty application for the effect after changes to conf. Simply exiting and opening does not work.

**Comment by hemanthrg** [comment]
I am still having an issue with my config file being ignored.
To test this, I changed the value of "confirm_os_window_close" from -1 to 0. I opened the config file with keyboard shortcut "Command + ,"

I have tried starting kitty from another terminal with /Applications/kitty.app/Contents/MacOS/kitty -c ~/.config/kitty/kitty.conf.

I have pressed "Command" + "," and made sure that config file name is "~/.config/kitty/kitty.conf". I have verified that this file exists and that both of those files are identical.

I have checked my environment variables and I know I don't have any XDG variables or the variable KITTY_CONFIG_DIRECTORY.

I have tried quitting and restarting my application. I have also restarted my computer.

I have tried to use the --debug-config option, but for some reason the terminal prints "Unknown option: --debug-config"
I think I have exhausted all my options and I am not sure how to resolve this issue. Any help would be appreciated.
